utils/sentry_client.py

```diff
@@ -0,0 +1,155 @@
+import requests
+import json
+
+class SentryClient:
+  ORGANIZATION = "sentry"
+
+  def __init__(self, host, token):
+    self.host = host
+    self.auth_token = token
+
+  def do_sentry_api_call(self, method, path, slugs, payload=None):
+    success_codes = {"get": 200, "post": 201, "delete": 204, "put": 200}
+    url = "%s/api/0/%s/" % (self.host, path)
+
+    if len(slugs) > 0:
+      url = "%s%s/" % (url, "/".join(slugs))
+
+    headers = {"Authorization": "Bearer " + self.auth_token}
+    call = getattr(requests, method, None)
+    if call == None:
+      print ("invalid http method %s" % method)
+      return None
+
+    response = call(url, headers=headers, json=payload)
+    if response.status_code != success_codes[method]:
+      print ("error: http %s returned %d trying to access %s" % (method, response.status_code, url))
+      return None
+
+    if response.status_code != 204:
+      return response.json()
+
+
+  # Organization functions
+  def get_organizations(self):
+    request = self.do_sentry_api_call("get", "organizations", [])
```

Comment:
nitpick: `request` is deceiving here, it would be better to use `response` or `response_json`

Reply:
Renamed